artemis-commons/src/main/java/org/apache/activemq/artemis/utils/FileUtil.java

```diff
@@ -63,10 +63,15 @@ public static final boolean deleteDirectory(final File directory) {
             attempts++;
          }
 
-         for (String file : files) {
-            File f = new File(directory, file);
-            if (!deleteDirectory(f)) {
-               logger.warn("Failed to clean up file: " + f.getAbsolutePath());
+         if (files == null) {
+            logger.warn("Could not list files to clean up in: " + directory.getAbsolutePath());
```

Comment:
@johnament I don't know, but FWIW the "Failed to clean up file" one that already was and still is there is not (assuming I understand what a coded log message is...)